script/02_DeployCLPoolManager.s.sol

```diff
@@ -5,24 +5,44 @@ import "forge-std/Script.sol";
 import {BaseScript} from "./BaseScript.sol";
 import {IVault} from "../src/interfaces/IVault.sol";
 import {CLPoolManager} from "../src/pool-cl/CLPoolManager.sol";
+import {Create3Factory} from "pancake-create3-factory/src/Create3Factory.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
 /**
+ * Step1:
  * forge script script/02_DeployCLPoolManager.s.sol:DeployCLPoolManagerScript -vvv \
  *     --rpc-url $RPC_URL \
  *     --broadcast \
- *     --slow \
- *     --verify
+ *     --slow
+ *
+ * Step2:
+ * forge verify-contract <address> CLPoolManager --watch --chain <chain_id>
```

Comment:
is this verified to work? was under assumption we need to pass in the constructor args

Reply:
Sorry, my fault. I just verified vault yestd and omitted those two actually have constructor args. Updated